clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1960,13 +1985,15 @@ class raw2trace_t : public trace_converter_t<raw2trace_t> {
 
         // Statistics on the processing.
         uint64 count_elided = 0;
+        uint64 count_duplicate_syscall = 0;
 
         uint64 cur_chunk_instr_count = 0;
         uint64 cur_chunk_ref_count = 0;
         memref_counter_t memref_counter;
         uint64 chunk_count_ = 0;
         uint64 last_timestamp_ = 0;
         uint last_cpu_ = 0;
+        app_pc last_pc_if_syscall_;
```

Comment:
How about =0 here, matching the surrounding lines and avoiding line 1945

Reply:
Done.